### PR TITLE
MS-6 add test file

### PR DESCRIPTION
**Pull request by SummitStha**
@kelsash @raeesaroj @ankurbelbase @santoshpurbey

### CODE REVIEW

File: mysite/test.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+testing branch
```
Comment (ankurbelbase) [inline mysite/test.txt]: Capitalize "testing"

Comment (AashishGrg) [inline mysite/test.txt]: Would You please Capitalize testing??

**Comment by AashishGrg** [comment]
:+1:

File: mysite/test.txt (hunk 1 above)
Comment (SummitStha) [reply mysite/test.txt]: :ok:

Comment (SummitStha) [reply mysite/test.txt]: ok thanks for the advice

**Comment by ankurbelbase** [comment]
👍

**Pull request merged**